src/scripts/deleteHelper.js

```diff
@@ -9,17 +9,37 @@ function alertText(options) {
     return alert(options);
 }
 
+function getDeleteText(item) {
```

Comment:
renaming to something like `getConfirmContent`/`getDeletionConfirmContent` and returning `title`, `text`, `confirmText`, as well as `primary` would go a long way into reducing excess code within `deleteItem` as you can just call this function directly below e.g. `return confirm(getConfirmContent(item))`

Reply:
Not sure if it is better especially when there it is not a TS file, but I dont mind applying your suggestion. Done.